uk.co.compucorp.civicrm.hrleaveandabsences/scss/components/_leave-calendar.scss

```diff
@@ -54,6 +54,12 @@ $chr-leave-calendar-legend-size:                  20px;
     height: 1.2em;
   }
 
+  > .chr_leave-calendar__time-unit {
+    font-size: 7px;
```

Comment:
In most cases we use variables for font sizes. Can you check defined variables  for `7px` font size?

Reply:
I checked, there were none:
https://github.com/compucorp/org.civicrm.shoreditch/blob/staging/scss/bootstrap/overrides/_variables.scss

I didn't add them because this is a very specific case, not sure if this is going to be repeated at all throughout the app